### ISSUE TITLE
Do not allow upload of the export.osm file task bbox into OSM.

### ISSUE THREAD

**Issue by maning** [hotosm/osm-tasking-manager#126]
I'm not sure how to do this but during Typhoon Bopha mapping, one user unintentionally uploaded the `export.osm`.  See this changeset for reference: http://www.openstreetmap.org/browse/changeset/14507926 . The actual bbox way is now deleted.

The osm xml of the export.osm file have the upload=false directive.  But users can still upload the file, in this case, using JOSM.
`<osm version="0.5" generator="HOT Tasking Manager" upload="false">`

**Comment by DmitriGoldring** [comment]
There is an undocumented feature of JOSM that will silently drop elements with the tag "josm/ignore" (presumably set to some value like 'true'). (See http://josm.openstreetmap.de/ticket/5307 or org.openstreetmap.josm.data.APIDataSet.java for details.) Adding this to the bbox way in export.osm would at least prevent the bbox from being uploaded. Unfortunately, if someone is using export.osm in another tool then they'll still get a bonus way, but now with added attribute.

**Comment by pgiraud** [comment]
Thanks @DmitriGoldring.
I would be happy to know why `ignore="false"` doesn't work anymore though. Are users doing something specific?

http://josm.openstreetmap.de/wiki/Changelog#a2012-03-07:Stablereleaser5047

**Comment by DmitriGoldring** [comment]
Yes, the export="false" triggers a warning but does not prohibit the upload.  The message reads: "Sending data from this layer is strongly discouraged." 
I did it once myself before I understood what the warning was trying to say. I did not have "Download as new layer" checked when I grabbed the OSM data and my changes got made on the same layer as the bounding box. I 'knew' that my changes were ok so I continued. Fortunately I reviewed the map and immediately realized the bounding box wasn't supposed to be there.
